ElectronicObserver/Data/MissionData.cs

```diff
@@ -30,7 +30,10 @@ public class MissionData : APIWrapper, IIdentifiable
 	/// 海域カテゴリID
 	/// </summary>
 	public int MapAreaID => (int)RawData.api_maparea_id;
-
```

Comment:
Make sure to have a line of space between properties if they have documentation. Space can be ignored if you have a lot of simple properties that only hold data.